javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/config/ConfigPropertiesBridge.java

```diff
@@ -111,9 +113,26 @@ public Map<String, String> getMap(String name, Map<String, String> defaultValue)
 
   @Nullable
   @Override
-  public DeclarativeConfigProperties getDeclarativeConfig(String instrumentationName) {
-    return configProvider != null
-        ? InstrumentationConfigUtil.javaInstrumentationConfig(configProvider, instrumentationName)
-        : null;
+  public DeclarativeConfigProperties getDeclarativeConfig(String node) {
+    if (configProvider == null) {
+      // declarative config is not supported at all
```

Comment:
what does `not supported` mean in this context?

Reply:
should have been `not used`